interpreter/config.yaml

```diff
@@ -14,4 +14,5 @@ system_message: |
   You are capable of **any** task.
 local: false
 model: "gpt-4"
-temperature: 0
\ No newline at end of file
+temperature: 0
+integrations: [helicone]
```

Comment:
This might be good to put in an example in the docs, but we won’t want to enable this by default in the core `config.yaml`.